lib/ansible/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -30,17 +30,21 @@
     aliases: ["name"]
   record_type:
     description:
-    - The type of DNS record name
-    - Currently, 'A', 'AAAA', 'PTR' and 'TXT' (added in 2.5) are supported
+    - The type of DNS record name.
+    - Currently, 'A', 'AAAA', 'A6', 'CNAME', 'DNAME', 'PTR' and 'TXT' are supported.
+    - "'A6', 'CNAME', 'DNAME' and 'TXT' are added in version 2.5."
     required: false
     default: 'A'
-    choices: ['A', 'AAAA', 'PTR', 'TXT']
+    choices: ['A', 'AAAA', 'A6', 'CNAME', 'DNAME', 'PTR', 'TXT']
```

Comment:
To what extent, if any, are A6 records used?

Reply:
Not much. Was previously used for/with 'AAAA' record. We can remove this option if required.

Reply:
The impression I have gotten is that A6 records aren't used at all these days. There is also the following informational [RFC 6563: Moving A6 to Historic Status](https://tools.ietf.org/html/rfc6563)

Hence I believe that listing A6 records only has the potential to cause confusion.

Unless I'm wrong, and there actually are people who use A6 records today.

Reply:
@andreaso Sorry, I didn't mean for it to get merged without resolving your comment!

Personally, I see value in allowing A6, simply because the FreeIPA API allows it. We may have a user who needs an A6 record, even if it's just for testing. But I could probably easily be convinced otherwise as well. ;-)

Reply:
Not something I feel strongly enough to argue about.

While I still think it would be slightly preferable to not list the A6 record type I do agree that the record type being supported by the FreeIPA API is a perfectly valid reason for Ansible to too support it.

Reply:
Let me know if you want me to revert A6 change.